client/app/states/administration/profiles/profiles.html

```diff
@@ -2,13 +2,13 @@
   <span>
     <ol class="breadcrumb">
       <li>
-        <a ui-sref="designer"> <span class="fa fa-angle-double-left"/>&nbsp;{{'Designer'|translate}}</a>
+        {{'Administration'|translate}}
       </li>
       <li class="active"><strong>{{vm.title}}</strong>
```

Comment:
The translation is done in the controller: https://github.com/ManageIQ/manageiq-ui-self_service/pull/168/files#diff-2d515a73391624ceaca86937ab21e654L37